metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
```

Comment:
I mean, those are comparable to `potential_temperature`, and shorter than `equivalent_potential_temperature`.

Almost think we need to write down our design philosophy for consistency...aka "the zen of MetPy"